index.bs

```diff
@@ -46,26 +46,25 @@ div.head h1 { clear: left; }
 # Introduction # {#introduction}
 <!-- ============================================================ -->
 
-This specification describes a font enumeration API for web browsers which may, optionally, allow users to grant access to the full set of available system fonts. For each font, low-level (byte-oriented) access to an SFNT [[!SFNT]] container that includes full font data.
+This specification describes a font enumeration API for web browsers which may, optionally, allow users to grant access to the full set of available system fonts. For each font, low-level (byte-oriented) access to an SFNT [[!SFNT]] container provides full font data.
 
 Web developers historically lack anything more than heuristic information about which local fonts are available for use in styling page content. Web developers often include complex lists of `font-family` values in their CSS to control font fallback in a heuristic way. Generating good fallbacks is such a complex task for designers that tools have been built to help "eyeball" likely-available local matches.
 
 Font enumeration helps by enabling:
 
-* Logging of likely-available fonts to improve server-side font rule generation.
-* Scripts to generate style rules based on "similar" local fonts, perhaps saving a download.
-* Improving styling options for user-generated content, allowing the generation of style rules via more expressive font selection menus.
+* Improving styling options for user-generated content.
+* Matching fonts declared by existing content.
```

Comment:
"Font enumeration helps: ... Improve styling options ... Match fonts" or similar so you don't have "... enabling improving"